routers/router.go

```diff
@@ -12,15 +12,16 @@ func init() {
 	// Dashboard
 	web.Router("/dashboard", &controllers.DashboardController{}, "get:Index")
 
+	// Scraper
+	web.Router("/dashboard/search_keyword", &controllers.ScraperController{}, "post:TextSearch")
+	web.Router("/dashboard/search_keyword_csv", &controllers.ScraperController{}, "post:FileSearch")
```

Comment:
I recommend the following naming to keep the URL concise and more user friendly:

```suggestion
	web.Router("/dashboard/search", &controllers.ScraperController{}, "post:TextSearch")
	web.Router("/dashboard/upload", &controllers.ScraperController{}, "post:FileSearch")
```